### PR TITLE
First version of the Imgix provider

### PR DESCRIPTION
**Pull request by p-jankowiak-mrx**

### CODE REVIEW

File: tools/go.mod [hunk 1]
```diff
@@ -0,0 +1,5 @@
+module tools
+
+go 1.15
+
+require github.com/bflad/tfproviderdocs v0.8.0 // indirect
```
Comment (0mk1) [inline tools/go.mod]: **suggestion:** Could we make those generated docs look the same as AWS ones? Also I see you use here `bflad/tfproviderdocs` but in docs you have:  `# generated by https://github.com/hashicorp/terraform-plugin-docs`.

File: test_infra/main.tf [hunk 2]
```diff
@@ -0,0 +1,80 @@
+terraform {
```
Comment (0mk1) [inline test_infra/main.tf:1]: **suggestion:** @p-jankowiak-mrx How timeconsuming could be adding test suite for provider?  @timwardle This project could be developed by community, and more features and more fixes/bugs could make regression testing time consuming and not feasible. Also IMHO introducing new people to make changes here would be lot easier with unit tests of some sort. If that is not a best decision here for various reasons, then at least we should have some `Contribute` docs.

**Review by 0mk1 — CHANGES_REQUESTED** [review]

File: test_infra/main.tf (hunk 2 above)
Comment (p-jankowiak-mrx) [reply test_infra/main.tf:1]: I added few tests I think cover most of the simple cases. If we have more time we can develop it further

File: tools/go.mod (hunk 1 above)
Comment (p-jankowiak-mrx) [reply tools/go.mod]: This package is indeed wrongly added, I corrected that. Regarding look of the docs I think AWS is using some magic and I wanted to implement docs ASAP and following terraform guides. Maybe in the future they will improve feel of the generated docs.

**Comment by emhagman** [comment]
@p-jankowiak-mrx @timwardle I wasn't sure where to post this but I noticed this repo is missing a `LICENSE` file which technically means no one is allowed to use this source code. Is that intentional? I'd love to fork this and update it to work with `darwin/arm64` but technically can't until there is a `LICENSE` file.

If not, do you mind adding a license by adding a `LICENSE` text file in the root of the project? You can choose one here https://choosealicense.com/

**Comment by emhagman** [comment]
cc @0mk1 who suggested changes originally

**Comment by emhagman** [comment]
@p-jankowiak-mrx @timwardle @0mk1 any thoughts???

**Pull request merged**